### PR TITLE
feat(autonomous_emergency_braking): add markers showing aeb convex hull polygons for debugging purposes

### PR DESCRIPTION
**Pull request by danielsanchezaran**
## Description
This PR adds a debug marker for the convex hulls created by the AEB, which can help debugging and tuning point cloud parameters for the module. 

Example
cap- 2024-09-12-17-53-18
cap- 2024-09-12-17-53-42
cap- 2024-09-12-17-53-49
Screenshot from 2024-09-12 18-18-33

## Related links

**Parent Issue:**

- Link

## How was this PR tested?
PSim
## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/node.hpp [hunk 1]
```diff
@@ -475,18 +478,26 @@ class AEB : public rclcpp::Node
...
     const rclcpp::Time & current_time, const Path & path, const std::vector<Polygon2d> & polygons,
     const std::vector<ObjectData> & objects, const std::optional<ObjectData> & closest_object,
-    const double color_r, const double color_g, const double color_b, const double color_a,
-    const std::string & ns, MarkerArray & debug_markers);
+    const colorTuple & debug_colors, const std::string & ns, MarkerArray & debug_markers);
+
+  /**
+   * @brief Add a marker for debugging
+   * @param current_time Current time
+   * @param
```
Comment (kyoichi-sugahara) [inline control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/node.hpp:493]: :angry:

File: control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/utils.hpp [hunk 2]
```diff
@@ -83,9 +87,24 @@ Polygon2d convertBoundingBoxObjectToGeometryPolygon(
  */
 Polygon2d convertObjToPolygon(const PredictedObject & obj);
 
+/**
+ * @brief Get the predicted object's shape as a geometry polygon
```
Comment (kyoichi-sugahara) [inline control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/utils.hpp:91]: :angry:

**Review by kyoichi-sugahara — APPROVED** [review]
LGTM (after fixing the doxygen description!)

File: control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/utils.hpp (hunk 2 above)
Comment (danielsanchezaran) [reply control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/utils.hpp]: fixed!

File: control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/node.hpp (hunk 1 above)
Comment (danielsanchezaran) [reply control/autoware_autonomous_emergency_braking/include/autoware/autonomous_emergency_braking/node.hpp]: fixed

**Pull request merged**